CHANGELOG.md

```diff
@@ -6,27 +6,33 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
-## [1.2.0] 2020-01-08
+## [v1.2.0] 2020-01-08
```

Comment:
I agree with fixing this but the auto generation we use now does not prepend `v` in these links

Reply:
True, thanks. Will remove

Reply:
Done. PTAL